.eslintrc

```diff
@@ -16,6 +16,7 @@
     "curly": [1, "all"],
     "eqeqeq": [1, "smart"],
     "import/no-default-export": 2,
+    "import/no-named-as-default": 0,
```

Comment:
This is dangerous!!!

For
```
export const MyComponent = () => <div />;

export default connect(MyComponent, ...);
```

and 
```
import MyComponent from "./MyComponent";
```

It marks this import as a warning and suggests to import it as
```
import { MyComponent } from "./MyComponent"; 
```

Reply:
Should not be enabled until we get rid of all default exports!